### PR TITLE
feat: add index to SliderOnChangeCallback

### PR DESCRIPTION
**Pull request by KiwiKilian**
This helps to identify, which thumb was modified.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -31,12 +34,12 @@ export type SliderProps = {
     onSlidingStart?: SliderOnChangeCallback;
     onValueChange?: SliderOnChangeCallback;
     renderAboveThumbComponent?: (
-        value: number,
         index: number,
```
Comment (KiwiKilian) [inline src/types.ts:37]: These were mixed up, see how they are called: https://github.com/miblanchard/react-native-slider/blob/a4fe59d52460ccbd064c000c3df5d3f424d731cb/src/index.tsx#L759

File: src/types.ts [hunk 2]
```diff
@@ -31,12 +34,12 @@ export type SliderProps = {
...
+        value: number,
     ) => React.ReactNode;
     renderBelowThumbComponent?: (
-        value: number,
         index: number,
```
Comment (KiwiKilian) [inline src/types.ts:41]: Same as above: https://github.com/miblanchard/react-native-slider/blob/a4fe59d52460ccbd064c000c3df5d3f424d731cb/src/index.tsx#L895

**Review by miblanchard — APPROVED** [review]
Thank you very much 😄

**Comment by KiwiKilian** [comment]
@miblanchard anychance getting this released? Thanks! 🙏

**Pull request merged**